mail/rspamd/Portfile

```diff
@@ -2,78 +2,98 @@
 
 PortSystem          1.0
 PortGroup           cmake 1.0
```

Comment:
This port needs to switch to the cmake 1.1 portgroup. 

see https://trac.macports.org/ticket/47197.

Reply:
Thanks. I hope if I hit the `Resolve` button that just means your edits get added and that's all I do, right?

Reply:
Okay, ran into this: https://trac.macports.org/ticket/47197

Added this makes it compile.

```
cmake.out_of_source  no
```

Reply:
> Thanks. I hope if I hit the `Resolve` button that just means your edits get added and that's all I do, right?



> Thanks. I hope if I hit the `Resolve` button that just means your edits get added and that's all I do, right?

No. You have to make the requested changes yourself.